gems/aws-partitions/partitions.json

```diff
@@ -6,6 +6,12 @@
       "signatureVersions" : [ "v4" ]
     },
     "dnsSuffix" : "amazonaws.com",
+    "dualstackDefaults" : {
```

Comment:
This file (as is) should not be committed. Having it here to run tests. Prior to release we will get an official copy.